sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -101,9 +103,11 @@ export async function saveFakeApplicationDisbursements(
     student?: Student;
     msfaaNumber?: MSFAANumber;
     program?: EducationProgram;
+    programYear?: ProgramYear;
```

Comment:
can you please explain where are we using this in the code?

Reply:
The entire `relations` is passed to the `saveFakeApplication` which can potentially use it.